src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -77,4 +79,20 @@
     <bean id="arEmailService" parent="arEmailService-parentBean"
         class="edu.cornell.kfs.module.ar.service.impl.CuAREmailServiceImpl" />
 
+    <bean id="cuVerifyBillingFrequencyService" parent="verifyBillingFrequencyService"/>
+    <bean id="verifyBillingFrequencyService-parentBean"
+          class="edu.cornell.kfs.module.ar.batch.service.impl.CuVerifyBillingFrequencyServiceImpl"
+          p:universityDateService-ref="universityDateService"
+          p:businessObjectService-ref="businessObjectService"
+          p:accountingPeriodService-ref="accountingPeriodService"
+          p:dateTimeService-ref="dateTimeService"
+          p:cuContractsGrantsBillingUtilityService-ref="cuContractsGrantsBillingUtilityService"/>
+
+    <bean id="cuContractsGrantsBillingUtilityService" parent="contractsGrantsBillingUtilityService"/>
+    <bean id="contractsGrantsBillingUtilityService-parentBean"
```

Comment:
Shouldn't this override the bean "contractsGrantsBillingUtilityService" instead of "contractsGrantsBillingUtilityService-parentBean"?